dotCMS/src/main/java/com/dotcms/util/CollectionsUtils.java

```diff
@@ -926,7 +925,27 @@
 	    return new ImmutableListCollector<>();
     }
 
-	private static class ImmutableListCollector<T> implements Collector<T, ImmutableList.Builder<T>, ImmutableList<T>> {
+    /**
```

Comment:
While this method looks ok, as a general rule, I am not a fan of writing shallow custom wrappers on top of what apis provide.  It seems like code sprawl.  Do we need to maintain boilerplate code like this?  Why not just use a lamda filter ... and `.findFirst()` which is going to be better tested?

Reply:
you right, removed